src/Nancy/Json/Simple/SimpleJson.cs

```diff
@@ -1237,6 +1238,25 @@ interface IJsonSerializerStrategy
         string MapDictionaryKeyToFieldName(string stringKey);
     }
 
+#if SIMPLE_JSON_INTERNAL

```

Comment:
> I just noticed it wasn't included like on the existing classes

You've completely lost me. What existing classes? Assume I get _nothing_ about the context that this is discussed (because it's the truth! 😄)